Apps/WebClient/src/ClientApp/src/components/services/OrganDonorDetailsCard.vue

```diff
@@ -0,0 +1,183 @@
+import Vue from "vue";
+import { Component, Prop, Ref } from "vue-property-decorator";
+import { Action, Getter } from "vuex-class";
+
+import MessageModalComponent from "@/components/modal/MessageModalComponent.vue";
+import PatientData, {
+    HealthOptionTypes,
+    OrganDonorRegistrationData,
+    PatientDataFile,
+    PatientHealthOptions,
+} from "@/models/patientData";
+import container from "@/plugins/container";
+import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
+import { ILogger } from "@/services/interfaces";
+import SnowPlow from "@/utility/snowPlow";
+
+library.add(faCircleInfo, faDownload);
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const options: any = {
+    components: { MessageModalComponent },
+};
+
+@Component(options)
+export default class OrganDonorDetailsCard extends Vue {
+    @Prop({ required: true })
+    hdid!: string;
+
+    @Getter("patientData", { namespace: "patientData" })
+    getPatientData!: (hdid: string) => PatientData;
+
+    @Action("retrievePatientDataFile", { namespace: "patientData" })
+    getPatientDataFile!: (params: {
+        fileId: string;
+        hdid: string;
+    }) => Promise<PatientDataFile>;
+
+    @Ref("sensitiveDocumentModal")
+    readonly sensitiveDocumentModal!: MessageModalComponent;
+
+    logger!: ILogger;
+
+    get registrationData(): OrganDonorRegistrationData | undefined {
+        return this.patientData.items.find(
+            (ho: PatientHealthOptions) =>
+                ho.type === HealthOptionTypes.OrganDonorRegistrationData
+        ) as OrganDonorRegistrationData;
+    }
+
+    get patientData(): PatientData {
+        return this.getPatientData(this.hdid);
+    }
+
+    private created(): void {
+        this.logger = container.get<ILogger>(SERVICE_IDENTIFIER.Logger);
+    }
+
+    private showConfirmationModal(): void {
+        this.sensitiveDocumentModal.showModal();
+    }
+
+    private getDecisionFile(): void {
+        if (this.registrationData && this.registrationData.registrationFileId) {
+            SnowPlow.trackEvent({
+                action: "download_report",
+                text: "Organ Donor",
+            });
+            this.getPatientDataFile({
+                fileId: this.registrationData.registrationFileId,
+                hdid: this.hdid,
+            })
+                .then(
+                    (patientFile: PatientDataFile) =>
+                        new Blob(patientFile.content, {
+                            type: patientFile.contentType,
+                        })
+                )
+                .then((blob) => saveAs(blob, `Organ_Donor_Registration.pdf`))
+                .catch((err) => this.logger.error(err));
+        }
+    }
+}
+</script>
+
+<template>
+    <b-card
+        container
+        class="hg-card-button h-100 w-100 d-flex flex-column align-content-start text-left rounded shadow border-0"
+        header-class="border-0 bg-transparent d-flex"
+        v-bind="$attrs"
+    >
```

Comment:
I recommend instead using our HgCardButton component and just updating so doesn't have a click action.

We should probably also reduce the amount of whitespace between the fields (Status/Decision/Register) to match the spacing we use between fields in the timeline.

Reply:
I wanted to get a response from Mandad on this one, but he wasn't available at the time, I went to mimic the mocks, but I'll take a look at the HgButtonCard comment. I'll resolve this tonight, I have to go out for a bit.

Reply:
I'll take a look & see if I can get Mandad's feedback.